Android/Nothing/fastlane/report.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="fastlane.lanes">
+    
+    
+    
+      
+      <testcase classname="fastlane.lanes" name="0: default_platform" time="0.000562">
+        
+      </testcase>
+    
```

Comment:
Wild empty line

Reply:
Same, file Is generated, no need for it at repo